tomviz/DoubleSliderWidget.cxx

```diff
@@ -11,6 +11,8 @@
 #include <QHBoxLayout>
 #include <QSlider>
 
+#define DEFAULT_DOUBLE_PRECISION_VALUE 16
```

Comment:
I think we should remove the vtkPVConfig.h include as I think that is only there for this variable. Also, there is only one call site, let's just put the number in there - it feels over-engineered to use a define for a single call site. I wonder if 8 might also be enough, but happy to go either way there.

Reply:
Done and done. I also changed the precision to 8.